code/game/area/areas.dm

```diff
@@ -67,6 +67,8 @@
 	var/moving = FALSE
 	/// "Haunted" areas such as the morgue and chapel are easier to boo. Because flavor.
 	var/is_haunted = FALSE
+	///This variable, normaly set to true by a gravity grenade, means it checks this variable before it checks if a gravity generator exists on the level, meaning you can have a no gravity room on the same level as a generator.
+	var/special_gravity = FALSE
```

Comment:
Very non-descriptive variable name.

Reply:
Changed to the one you suggested on discord